res/controllers/midi-components-0.0.js

```diff
@@ -198,7 +198,7 @@
                     if (this.isLongPressed) {
                         this.inToggle();
                     }
-                    if (this.longPressTimer !== 0) {
+                    if (this.longPressTimer !== undefined && this.longPressTimer !== 0) {
```

Comment:
Do I dare suggesting checking for a falsy value?

```suggestion
                    if (this.longPressTimer) {
```

Reply:
I'm not sure if there could be any side effect here

Reply:
generally not a fan of this pattern because JS implicit conversions to bool are somewhat of a footgun.

Reply:
Yeah I was expecting this feedback. My main frustration is the case it wouldn't handle something like `null`, meaning that once the timer as run, you are expected to set it to `0` or `undefined` which can feel a bit weird.

Anyway, I'm nitpicking here, happy to go with the current solution!

Reply:
yeah, `null` is also weird. Ideally we'd introduce a named constant (see request below).